share/instruments/instruments.xml

```diff
@@ -7319,6 +7318,220 @@
                   <barlineSpan>1</barlineSpan>
                   <drumset>1</drumset>
                   <singleNoteDynamics>0</singleNoteDynamics>
+                  <Drum pitch="35"> <!--Acoustic Bass Drum-->
+                        <head>normal</head>
+                        <line>8</line>
+                        <voice>1</voice>
+                        <name>Acoustic Bass Drum</name>
+                        <stem>2</stem>
+                        <panelRow>2</panelRow>
+                        <panelColumn>1</panelColumn>
+                  </Drum>
+                  <Drum pitch="36"> <!--Electric Bass Drum-->
+                        <head>normal</head>
+                        <line>7</line>
+                        <voice>1</voice>
+                        <name>Bass Drum 1</name>
+                        <stem>2</stem>
+                        <shortcut>B</shortcut>
+                        <panelRow>2</panelRow>
+                        <panelColumn>0</panelColumn>
+                  </Drum>
+                  <Drum pitch="37"> <!--Side Stick-->
+                        <head>slashed1</head>
+                        <line>3</line>
+                        <voice>0</voice>
+                        <name>Side Stick</name>
+                        <stem>1</stem>
+                        <panelRow>1</panelRow>
+                        <panelColumn>1</panelColumn>
+                  </Drum>
+                  <Drum pitch="38"> <!--Acoustic Snare-->
+                        <head>normal</head>
+                        <line>3</line>
+                        <voice>0</voice>
+                        <name>Acoustic Snare</name>
+                        <stem>1</stem>
+                        <shortcut>A</shortcut>
+                        <panelRow>1</panelRow>
+                        <panelColumn>0</panelColumn>
+                  </Drum>
+                  <Drum pitch="40"> <!--Electric Snare-->
+                        <head>slash</head>
+                        <line>3</line>
+                        <voice>0</voice>
+                        <name>Electric Snare</name>
+                        <stem>1</stem>
+                        <panelRow>2</panelRow>
+                        <panelColumn>6</panelColumn>
+                  </Drum>
+                  <Drum pitch="41"> <!--Low Floor Tom-->
+                        <head>normal</head>
+                        <line>6</line>
+                        <voice>0</voice>
+                        <name>Low Floor Tom</name>
+                        <stem>1</stem>
+                        <panelRow>1</panelRow>
+                        <panelColumn>7</panelColumn>
+                  </Drum>
+                  <Drum pitch="42"> <!--Closed Hi-hat-->
+                        <head>cross</head>
+                        <line>-1</line>
+                        <voice>0</voice>
+                        <name>Closed Hi-Hat</name>
```

Comment:
`Hi-Hat` here...

Reply:
Are you sure about the "Hi-Hat"? There are other places where it is called "Hi-hat":
https://github.com/musescore/MuseScore/blob/5962d4663b0af706eb9874f1d7f6118e3ccb0a86/share/instruments/instruments.xml#L8385-L8388